lib/components/chat_history/message.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:firebase_remote_config/firebase_remote_config.dart';
+// import 'package:firebase_remote_config/firebase_remote_config.dart';
```

Comment:
Remote config code can be removed from here since it is no longer used for these events.